src/test/java/gumtree/spoon/AstComparatorTest.java

```diff
@@ -1790,14 +1786,11 @@ public void testExtendsGenerics1() throws Exception {
 		result.debugInformation();
 
 		assertEquals(1, actions.size());
-		assertTrue(result.containsOperation(OperationKind.Update, "SUPER_TYPE", "SuperClass<One>"));
+		assertTrue(result.containsOperation(OperationKind.Update, "TypeReference", "One"));
```

Comment:
I further fine-tuned it as `SUPER_TYPE` is not updated but `One` is, which is an instance of `CtTypeReference`.